MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/View/ComposeContentView.swift

```diff
@@ -23,6 +23,11 @@ public struct ComposeContentView: View {
     
     @ObservedObject var viewModel: ComposeContentViewModel
     
+    @State private var randomContentInputPlaceholder = [
+        L10n.Scene.Compose.contentInputPlaceholder,
+        L10n.Scene.Compose.contentInputPlaceholder2,
+        L10n.Scene.Compose.contentInputPlaceholder3,
+    ].randomElement()
```

Comment:
You should be safe to force unwrap this since `randomElement` only returns nil if the collection is empty ([docs](https://developer.apple.com/documentation/swift/array/randomelement(using:))).

Reply:
Yup, that makes sense. Done!